GVRf/Sample/gvrbulletsample/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="com.android.ide.eclipse.adt.ANDROID_FRAMEWORK"/>
+	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.DEPENDENCIES"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="gen"/>
+	<classpathentry exported="true" kind="con" path="com.android.ide.eclipse.adt.LIBRARIES"/>
+	<classpathentry kind="lib" path="C:/Users/deepak.rawat/Desktop/Source/GearVRf/GVRf/Sample/gvrbulletsample/libs/bulletjniframework.jar"/>
+	<classpathentry kind="output" path="bin/classes"/>
```

Comment:
"C:/Users/deepak.rawat/Desktop/Source/GearVRf/GVRf/Sample/gvrbulletsample/libs/bulletjniframework.jar"
I am pretty sure that not every computer has this path.   :-)